### PR TITLE
Add masked icon and update manifest for pwa

### PR DESCRIPTION
**Pull request by kyleshay**
PWA builder was erroring explaining that a "maskable" icon was missing. 

https://web.dev/maskable-icon/
https://maskable.app/

Also semi related but the manifest icons can be svg and you can just provide multiple sizes in the string
```json
  {
    "src": "icon/hd_hi.ico",
    "sizes": "72x72 96x96 128x128 256x256"
  },
  {
    "src": "icon/hd_hi.svg",
    "sizes": "72x72"
  }
```
https://developer.mozilla.org/en-US/docs/Web/Manifest/icons

### CODE REVIEW

File: icons/build_icons.js [hunk 1]
```diff
@@ -31,6 +31,11 @@ for (const VERSION of ['release', 'beta', 'dev']) {
   execSync(
     `rsvg-convert -w 512 -h 512 -o "${VERSION}/android-chrome-512x512-${CACHEBREAKER}.png" "android-icon-${VERSION}.svg"`
   );
+  if (VERSION === 'release') {
+    execSync(
+      `rsvg-convert -w 512 -h 512 -b "#ee6d0d" -o "${VERSION}/android-chrome-mask-512x512-${CACHEBREAKER}.png" "android-icon-${VERSION}.svg"`
```
Comment (kyleshay) [inline icons/build_icons.js:36]: Could also switch on the VERSION and use a different background color for beta/dev?

File: src/app/App.tsx [hunk 2]
```diff
@@ -104,6 +104,7 @@ export default function App() {
                     'progress',
                     'records',
                     'optimizer',
+                    'loadouts',
```
Comment (kyleshay) [inline src/app/App.tsx:107]: makes it so you can go directly to `https://app.destinyitemmanger.com/loadouts`

File: src/manifest-webapp-6-2018.json [hunk 3]
```diff
@@ -11,10 +11,34 @@
       "src": "/android-chrome-512x512-6-2018.png",
       "sizes": "512x512",
       "type": "image/png"
+    },
+    {
+      "src": "/android-chrome-mask-512x512-6-2018.png",
+      "sizes": "512x512",
+      "type": "image/png",
+      "purpose": "maskable"
```
Comment (kyleshay) [inline src/manifest-webapp-6-2018.json:19]: the change so that we can get though PWA builder

File: src/manifest-webapp-6-2018.json [hunk 4]
```diff
@@ -11,10 +11,34 @@
...
+    },
+    {
+      "name": "Loadouts",
+      "url": "/loadouts",
+      "description": "Create & share Loadouts"
+    },
+    {
+      "name": "Settings",
+      "url": "/settings"
     }
```
Comment (kyleshay) [inline src/manifest-webapp-6-2018.json:36]: If you long-press an icon you can see secondary actions. Could be useful but I'm fine with removing them? Just a secondary way to quickly jump to see what Xur has, etc. Also a nice way to show off features in a secondary menu? These could also have their own custom icons. 

https://developer.mozilla.org/en-US/docs/Web/Manifest/shortcuts

File: src/manifest-webapp-6-2018.json [hunk 5]
```diff
@@ -11,10 +11,34 @@
...
   ],
   "theme_color": "black",
   "background_color": "black",
   "display": "standalone",
+  "orientation": "any",
+  "categories": ["games", "entertainment", "productivity", "utilities"],
```
Comment (kyleshay) [inline src/manifest-webapp-6-2018.json:42]: PWA / manifest checkers are happier when you give them more detail. Fine with cutting it out, not required.

File: icons/build_icons.js (hunk 1 above)
Comment (bhollis) [reply icons/build_icons.js:36]: I figure we'll only put one version in PWA builder?

File: src/manifest-webapp-6-2018.json (hunk 4 above)
Comment (bhollis) [reply src/manifest-webapp-6-2018.json:36]: I'm into it! I didn't know this existed. Wish we could localize it.

File: src/manifest-webapp-6-2018.json [hunk 6]
```diff
@@ -11,10 +11,34 @@
...
+      "sizes": "512x512",
+      "type": "image/png",
+      "purpose": "maskable"
+    }
+  ],
+  "shortcuts": [
+    {
+      "name": "Vendors",
+      "url": "/vendors",
+      "description": "View Vendors wares"
```
Comment (bhollis) [inline src/manifest-webapp-6-2018.json:26]:
```suggestion
      "description": "View vendors' wares"
```

File: src/manifest-webapp-6-2018.json [hunk 7]
```diff
@@ -11,10 +11,34 @@
...
+    },
+    {
+      "name": "Loadouts",
+      "url": "/loadouts",
+      "description": "Create & share Loadouts"
```
Comment (bhollis) [inline src/manifest-webapp-6-2018.json:31]:
```suggestion
      "description": "Create & share loadouts"
```

**Review by bhollis — APPROVED** [review]

File: src/manifest-webapp-6-2018.json (hunk 4 above)
Comment (kyleshay) [reply src/manifest-webapp-6-2018.json:36]: Localizing would be sweet. Looks like there might be a few ways to do it - not sure what that would look like app side though. 
https://stackoverflow.com/questions/57763393/localising-a-pwa-web-manifest

I think it's also mostly fairly straightforward to do this ourselves with the auto-generated apps. We can just add in the shortcuts ourselves (it's just pointing to a URL, and we handle all of the deeplinking already.) 

Might be good to just have it as-is for now though? We could maybe track hits to those endpoints from the apps to see if people are liking them? Would that be easy to do, would it help if I added a query param at the end to denote it's being hit from an app, or is that not needed?

Comment (kyleshay) [reply src/manifest-webapp-6-2018.json]: Would it be useful to add the other endpoints into records/progress/organizer/etc? wondering what the cut-off should be?

Comment (kyleshay) [reply src/manifest-webapp-6-2018.json]: Also, do we use a generic icon for "loadouts" or "vendors"? I could use the gear icon for settings.

Comment (delphiactual) [reply src/manifest-webapp-6-2018.json]: https://www.youtube.com/watch?v=baSiSIyTGSk&t=326s

**Comment by kyleshay** [comment]
I'll look into and land a follow-up for translation

File: src/manifest-webapp-6-2018.json (hunk 4 above)
Comment (kyleshay) [reply src/manifest-webapp-6-2018.json]: Ooh that helps thanks! I'll look into trying it

**Pull request merged**